CaloOnlineTools/HcalOnlineDb/test/template.py

```diff
@@ -1,4 +1,5 @@
 import FWCore.ParameterSet.Config as cms
+from Configuration.StandardSequences.Eras import eras
```

Comment:
Please do not add unnecessary imports. Instead, replace `Run2_2018` with `Run3` below, since you will be generating LUTs with the Run3 rather than 2018 detector in the future.

Reply:
done